### DISCUSSION TITLE
osu! AppImage Error: Disabling tablet support due to error: "DeviceIOException: Unable to retrieve device string at index 6: OK"

### DISCUSSION THREAD

**Discussion by DinoSour69** [ppy/osu#24531] [Q&A]
### Type

Other

### Bug description

Ubuntu 20.04

Error appears when trying to play the game with a Huion H610Pro Graphics Tablet. I thought opentabletdriver may have been clashing with osu! in some way but after closing osu, killing otd, then re-opening osu the issue still persists.

### Version

2023.812.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/12328951/database.log)
[input.log](https://github.com/ppy/osu/files/12328952/input.log)
[network.log](https://github.com/ppy/osu/files/12328953/network.log)
[performance.log](https://github.com/ppy/osu/files/12328954/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12328955/runtime.log)

**Comment by bdach** [comment]
This is something that should be reported to opentabletdriver.

Closest existing upstream issue is https://github.com/OpenTabletDriver/OpenTabletDriver/issues/2258.
